### PR TITLE
Update configuration.md for #185

### PR DESCRIPTION
**Pull request by dvershinin**
Mentions required config changes to actually enable TFO on supported systems

### CODE REVIEW

**Comment by Dridi** [comment]
LGTM, @daghf?

File: docs/configuration.md [hunk 1]
```diff
@@ -181,6 +181,15 @@ disabled. The recommended way to to select protocols is to use
 The following tokens are available for the `tls-protos` option:
 `SSLv3`, `TLSv1.0`, `TLSv1.1` and `TLSv1.2`.
 
+## TCP Fast Open Support
+
+TCP Fast Open saves up to one full round-trip time (RTT) over 
+the standard three-way connection handshake during a TCP session.
+
+Since kernel version 3.13, the TCP Fast Open is enabled by default.
+However, it must be explcitly enabled in the configuration file:
```
Comment (Dridi) [inline docs/configuration.md:190]: Here we should avoid to implicitly reference Linux as "the kernel".

Please use something more neutral like "If the system supports TCP Fast Open, Hitch has this support built in be default".

It's a bit confusing to say that it is enabled by default but needs to be enabled. But please come up with something better than my unsatisfying suggestion :)

Comment (dvershinin) [reply docs/configuration.md:190]:
```suggestion
On a system which supports TCP Fast Open, Hitch is able to reduce 
network latency with the following in the configuration file:
```
I really don't know? :-)

Comment (gquintard) [reply docs/configuration.md:190]: looks good

**Pull request merged**